lib/caseflow/service.rb

```diff
@@ -36,6 +36,12 @@ def get_appeals(user, additional_headers = {})
           {},
           additional_headers.merge('ssn' => user.ssn)
         )
+
+        appeals = response.body['data']
+        # Track null issue descriptions in appeals
+        # If we are seeing a lot of these, we will need to take further action
+        handle_appeals_with_null_issue_descriptions(user, appeals) if appeals.present?
```

Comment:
Our team is also a consumer of the Caseflow service but doesn't have any need to know about null issue descriptions. Given that this is a specific need for your team, was it considered to add this logging to the controller or class that calls this Caseflow service instead?

If keeping the code here, given that the new code isn't critical to returning the appeal data, to avoid any risk of an error in the new code throwing an exception, I'd like to suggest wrapping it in a `begin...rescue` block and not throwing an exception if an error is reached.

Reply:
The `get_appeals` method calls another class, `Caseflow::Responses::Caseflow` that returns the `body` and `status` of the response after validating the body against a JSON schema. However, in the case of `null` issue descriptions, the schema validation will fail, and thus an error will be raised before code in the controller can be executed. I will add in a `begin...rescue` block

Reply:
A `begin...rescue` block has been added, along with a corresponding unit test.

Reply:
I understand now. Thanks for explaining and for adding the `begin...rescue` block.